### PR TITLE
tests for auth

### PR DESCRIPTION
**Pull request by BingingWithSagar**

### CODE REVIEW

File: cypress/integration/login.spec.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+describe('Test For Login Page and Forgot Password', function() {
+	context('Login Page', function() {
+		beforeEach(function() {
+			cy.visit('localhost:3000/login');
+        })
+        it('test login no input fileds',function(){
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
```
Comment (pranavpandey1998official) [inline cypress/integration/login.spec.js:8]: we should test that it is still on login page , `cy.url().should('include', '/login')`

File: cypress/integration/login.spec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            cy.get('input[name="email"]').type('manupanday1998@gmail.com').should('have.value', 'manupanday1998@gmail.com')
+            cy.get('input[name="password"]').type('123456').should('have.value', '123456')
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
+        });
+        it('test login invalid email',function(){
+            cy.get('input[name="email"]').type('sagmail.com').should('have.value', 'sagmail.com')
+            cy.get('input[name="password"]').type('retro').should('have.value', 'retro')
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
```
Comment (pranavpandey1998official) [inline cypress/integration/login.spec.js:20]: here /login

File: cypress/integration/login.spec.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        })
+        it('test login no input fileds',function(){
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
+        });
+        it('test login valid email',function(){
+            cy.get('input[name="email"]').type('manupanday1998@gmail.com').should('have.value', 'manupanday1998@gmail.com')
+            cy.get('input[name="password"]').type('123456').should('have.value', '123456')
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
```
Comment (pranavpandey1998official) [inline cypress/integration/login.spec.js:14]: here /home

File: cypress/integration/login.spec.js (hunk 2 above)
Comment (BingingWithSagar) [reply cypress/integration/login.spec.js:20]: since i am already on /login using '/' gives the desired result,

Comment (BingingWithSagar) [reply cypress/integration/login.spec.js:20]: I will still resolve for more clarity

Comment (pranavpandey1998official) [reply cypress/integration/login.spec.js:20]: yeah but if website, routes to /home for because of some bug , this will pass but we want it to fail right

**Comment by BingingWithSagar** [comment]
61ecb44 is the last commit , following commits were by mistake, was trying some new functions

**Comment by BingingWithSagar** [comment]
-debug the login, using the test there are a few wrong messages
-signup should only allow strong password(acc. to our user story), please if possible make that change

**Comment by BingingWithSagar** [comment]
> Yes I fixed it , pull latest changes . also see how I wait for network request to complete cy.wait(1000) is never a good idea

i agree, it makes testing dependent on server response, i read up on aliasing, couldn't figure it out, your improvements will help, thank you

**Pull request merged**